.github/workflows/build.yml

```diff
@@ -25,10 +25,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
     - name: 'Install Dependencies'
       run: sudo apt-get update && make ciprepare
     - name: 'Run rust tests'
-      run: make --keep-going test
+      run: |
+        sudo apt install libssl-dev;
+        cargo install cargo-tarpaulin;
```

Comment:
The installation should be part of the `'Install Dependencies'` step.

Reply:
I can only agree with you but I need to make it work first. It does not work when I change the places.

Reply:
Sorry, that was only half of it anyway. The `libssel-dev` headers should be in that step. The cargo stuff should be part of the `ciprepare` target and wrapped in make; see `Makefile` for how it's done with other tools.

Also wrap the invocations in `make` in general. Let's see if that works then. :-)